probe/endpoint/procspy/proc_linux.go

```diff
@@ -23,6 +24,23 @@ var (
 	netNamespacePathSuffix = ""
 )
 
+type pidWalker struct {
+	walker      process.Walker
+	ticker      <-chan time.Time
```

Comment:
I would call this `tickc` just so there's no confusion between the time.Ticker and its underlying chan.